src/appkit.cs

```diff
@@ -1159,6 +1159,7 @@ interface NSBezierPath : NSSecureCoding, NSCopying {
 		[Export ("bezierPathWithRect:")]
 		NSBezierPath FromRect (CGRect rect);
 
+		[ThreadSafe]
```

Comment:
I don't want to over generalize but I think it's safe to assume other similar methods will be thread safe on move the attribute on the class level.

Even more for this case and the `NString` extension since I could find:

> Drawing Restrictions
> The Application Kit is generally thread-safe when drawing with its graphics functions and classes, including the NSBezierPath and NSString classes.
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Multithreading/ThreadSafetySummary/ThreadSafetySummary.html